install/docker/src_code_install.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+####### Declare environment variables
+CODE=/code
+BMI3D=$CODE/bmi3d ### Directory in which to install the bmi3d software
+USER=root  # We're in a docker container so root is safe 
+
+####### Download any src code
+git clone https://github.com/sgowda/plot $HOME/code/plotutil
+git clone https://github.com/sgowda/robotics_toolbox $HOME/code/robotics
+# pygame
+hg clone https://bitbucket.org/pygame/pygame $HOME/code/pygame
```

Comment:
is this pip installable now?

Reply:
Yep, good catch! Pygame was already in requirements.txt, removed the redundant download here